src/govuk/common.mjs

```diff
@@ -6,6 +6,10 @@ import './vendor/polyfills/Element/prototype/closest.mjs'
  * TODO: Ideally this would be a NodeList.prototype.forEach polyfill
  * This seems to fail in IE8, requires more investigation.
  * See: https://github.com/imagitama/nodelist-foreach-polyfill
+ *
+ * @param {NodeListOf<Element>} nodes - NodeList from querySelectorAll()
```

Comment:
That's just me checking where it's called and could only see `NodeListOf<Element>`

Suppose you could have `document.querySelector('#example').childNodes` as `NodeListOf<ChildNode>`, shall I put both in the JSDoc? It's to avoid linting errors in TypeScript projects complaining that types are incompatible

```suggestion
 * @param {NodeListOf<Element | ChildNode>} nodes - NodeList from querySelectorAll()
```